src/windows/main_window.py

```diff
@@ -1311,6 +1323,25 @@ def actionNextMarker_trigger(self, event):
             get_app().window.refreshFrameSignal.emit()
             get_app().window.propertyTableView.select_frame(frame_to_seek)
 
+            # Center the timeline on the marker
+            self.timeline.centerOnTime(closest_position)
+            
+    def actionCenterOnPlayhead_trigger(self, event):
+        log.info('actionCenterOnPlayhead_trigger')
+        
+        # Get player object
+        player = self.preview_thread.player
+        
+        # Calculate frames per second
+        fps = get_app().project.get("fps")
+        fps_float = float(fps["num"]) / float(fps["den"])
+        
+        # Calculate the playhead position in seconds
+        playheadPosition = player.Position() / fps_float
+        
+        # Center the timeline on the playhead position
+        self.timeline.centerOnTime(playheadPosition)
```

Comment:
It seems like there's too much math being done on the Python side here, when the JavaScript side already has the necessary information. 

It would be far more efficient to simply have a JavaScript function `$scope.centerOnPlayhead()`, and let it do all of those lookups. They're _easy_, from that side, as all of the data is basically right there. Then, the Python side just calls that with no arguments when necessary.

See `$scope.MovePlayhead` and `$scope.MovePlayheadToFrame` for examples of the necessary lookups.

Reply:
(Basically, on the JavaScript side, `$scope.centerOnPlayhead()` can just be...):
```js
$scope.centerOnPlayhead = function() {
  return $scope.centerOnTime($scope.project.playhead_position);
);
```

Reply:
I'm embarrassed to say that I spent 15 minutes figuring out how to implement centerOnPlayhead function because I didn't read your last comment...

I've never worked with JavaScript before.  Should every function have a return statement?  It works without it, but is it good practice to include one?

Reply:
Only if necessary. TBH the only reason I wrote that with a `return` was that I was sketching it out off the top of my head, and wasn't sure whether `centerOnTime()` itself returned anything. (Obvious in retrospect that it doesn't, but in the moment I think I just shrugged and went, "Well, if it does, then we should pass it along.") So I tossed it in there on the theory that it was safer than possibly losing a return value. :laughing: